### ISSUE TITLE
Pinterest share button is not working

### ISSUE THREAD

**Issue by moxatgithub** [MurhafSousli/ngx-sharebuttons#372]
#### I am submitting a

- [x] Bug Report

#### What is the expected behavior?
Pinterest share dialog should show

#### What is the current behavior?
Pinterest button share is not working. There's an error in core.

```
ShareButton.html:1 ERROR TypeError: Cannot read property 'then' of undefined
at ShareDirective.push../node_modules/@ngx-share/core/fesm5/ngx-share-core.js.ShareDirective.share (ngx-share-core.js:1447)
```

#### What are the steps to reproduce?
Enable pinterest button share

#### Which versions are you using for the following packages?

Angular: @angular/core ~7.2.0
Angular CDK: ^7.3.0 
Angular CLI: ~7.2.3
Angular-fontawesome: 0.3.0
Share buttons: ['facebook', 'twitter', 'email', 'pinterest']

**Comment by varunsahni003** [comment]
Yes, I am also getting same error for pinterest, while others are working fine

**Comment by MurhafSousli** [comment]
Ok, Pinterest button is working properly, this is not a bug!

The reason why you get the error is because you have not selected an image to pin!

There are 2 ways to get **Pinterest** button to work:

1 - Use the `[image]` to set the image URL directly.
2 - Use `[autoSetMeta]="true"` which sets the image parameter from the meta tags (ofcourse this requires having the meta tags in document head), example:

```html
<head>
  ...
  <meta property="og:image" content="http://ia.media-imdb.com/images/rock.jpg" />
</head>
```

Go to the demo https://murhafsousli.github.io/ngx-sharebuttons/#/share-buttons-component and toggle the `set auto meta tags` and try pinterest button again.

---

I should handle the error thrown with a warning message states the Pinterest have not get an image parameter
